docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
 
 These items represent Helm repositories, and can be either traditional Helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
 
+#### Add Custom Git Repositories
+
+Click the `Create` button and be sure to select the target `Git repository containing Helm chart...` to add a custom Git repository that contains your Helm charts or cluster template definitions. The branch name can be specified here if necessary, and if no branch name is provided the main branch will be used by default. At minimum a name and Git repository URL will be needed to finalize creation. An optional field for describing the chart repository is also available. Whenever you add a chart repository to Rancher, it will be available immediately.
+
+#### Add Custom Helm Chart Repositories
+
+A Helm chart repository is an HTTP server that houses one or more packaged charts. Any HTTP server that can serve YAML files and tar files and can answer GET requests can be used as a repository server.
+
+Helm comes with Helm serve, a built-in package server for developer testing. The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled or hosting custom chart repository server using open-source projects like [ChartMuseum](https://github.com/helm/chartmuseum).
```

Comment:
```suggestion
Helm comes with Helm serve, a built-in package server for developer testing. The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled, or hosting custom chart repository server using open-source projects such as [ChartMuseum](https://github.com/helm/chartmuseum).
```

I'm not sure how "Helm serve" should be formatted and search results conflate it with the command, `helm serve`. Are they the same thing? This page formats it with backticks: https://zeet.co/marketplace/community/helm-serve